src/transformers.rs

```diff
@@ -1,20 +1,30 @@
-use proc_macro2::TokenStream;
+use proc_macro2::{TokenStream, TokenTree};
 use quote::quote;
 use syn::{visit_mut::VisitMut, Expr, ExprBlock};
 
 /// Convert the body of the fuzz_target! macro into a test function
-fn to_test_fn(content_tokens: TokenStream) -> String {
+fn to_test_fn(content_tokens: TokenStream, input_identifier: TokenStream) -> String {
     let test_fn = quote! {
         #[test]
         fn test_something() {
-            let data = [];
+            let #input_identifier = [];
```

Comment:
need to add a comment to indicate this is a template,
something like
```rust
let #input_identifier = []; // this is a test template
```
so we can substitute recorded fuzz inputs to here